### DISCUSSION TITLE
Too much new lines

### DISCUSSION THREAD

**Discussion by ErykCh** [danny-avila/LibreChat#4512] [Q&A]
### What happened?

This is user prompt in edit format:

![image](https://github.com/user-attachments/assets/fc26e4f1-7aab-4be4-b727-d2726a355752)

and how this looks like in the chat history when I select cancel:

![image](https://github.com/user-attachments/assets/3d58a913-6893-4caf-8afc-2cc26d1ee76b)

### Steps to Reproduce

Add a prompt with an empty new line

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is the way markdown gets rendered based on your input text. You can disable it in the settings, you should see a difference:

![image](https://github.com/user-attachments/assets/3507db1e-b69c-4ad9-99a9-f8c84b4153fc)
